kedro/framework/cli/starters.py

```diff
@@ -481,6 +481,10 @@ def _convert_tool_names_to_numbers(selected_tools: str | None) -> str | None:
     """
     if selected_tools is None:
         return None
+    if selected_tools.lower() == "none":
+        return ""
+    if selected_tools.lower() == "all":
+        return ",".join(TOOLS_SHORTNAME_TO_NUMBER.values())
```

Comment:
Maybe we just need to update the TOOLS_SHORTNAME_TO_NUMBER dictionary? Put "none" and "all" there? Or it will break something else?